.env.example

```diff
@@ -13,3 +13,6 @@ COOKIE_SECRET=0PK8jAZ3BgnFWP8Tuvaw
 SENDGRID_API_KEY=
 MAILER_FROM_EMAIL=my-stuff@test.com
 MAILER_FROM_NAME=My Stuff
+SITE_PROTOCOL=http://
+SITE_DOMAIN=localhost
+SITE_PORT=:8080
```

Comment:
1. Site port is not `:8080`, it's `8080`
2. Better to make webpack dev server to use this port instead of it's default one